src/components/SubPage/convert-data.ts

```diff
@@ -92,6 +95,56 @@ export const unpackEditingDesignData = (editing_design?: {
     }
 };
 
+export const unpackGenomicCharacterization = (gc?: GenomicCharacterizationFrontMatter): UnpackedGenomicCharacterization | null => {
+    if (!gc) {
+        return null
+    }
+
+    const diagrams = unpackMultiImageDiagrams(gc.diagrams);
+    const amplifiedJunctionsData = {
+        caption: gc.junction_table_caption,
+        data: (gc.amplified_junctions ?? []).map((junction) => {
+            return {
+                editedGene: junction.edited_gene,
+                junction: junction.junction,
+                expectedSize: junction.expected_size,
+                confirmedSequence: junction.confirmed_sequence
+            }
+        })
+    }
+    const ddpcrData = {
+        caption: gc.ddpcr_caption,
+        data: (gc.ddpcr ?? []).map((ddpcr) => {
+            return {
+                tag: ddpcr.tag,
+                clone: ddpcr.clone,
+                fpRatio: ddpcr.fp_ratio,
+                plasmid: ddpcr.plasmid
+            }
+        })
+    }
+    const crRnaOffTargetsData = {
+        caption: gc.off_targets_caption,
+        data: (gc.cr_rna_off_targets ?? []).map((offTarget) => {
+            return {
+                clonesAnalyzed: offTarget.clones_analyzed,
+                offTargetsSequenced: offTarget.off_targets_sequenced_per_clone,
+                totalSitesSequenced: offTarget.total_sites_sequenced,
+                mutationsIdentified: offTarget.mutations_identified
+            }
+        })
+    }
+    const data = {
+        amplifiedJunctions: hasTableData(amplifiedJunctionsData) ? amplifiedJunctionsData : undefined,
+        ddpcr: hasTableData(ddpcrData) ? ddpcrData : undefined,
+        crRnaOffTargets: hasTableData(crRnaOffTargetsData) ? crRnaOffTargetsData : undefined,
+        diagrams: nonEmptyArray(diagrams) ? diagrams : undefined,
```

Comment:
take it or leave it, but I would maybe make a default `emptySubpageData` object that has all the keys you need, `data: [], caption: ""` and then fill it in if the data is there. That way you don't have as many `if exists` checks.